extensions/search-mdn/package.json

```
@@ -24,7 +24,30 @@
       "title": "Search Web Docs",
       "subtitle": "MDN",
       "description": "Search MDN Web Docs",
-      "mode": "view"
+      "mode": "view",
+      "preferences": [
+        {
+          "type": "dropdown",
+          "title": "Preferred Action",
+          "data": [
+            {
+              "title": "Read Document",
+              "value": "read"
+            },
+            {
+              "title": "Open in Browser",
+              "value": "open"
+            },
+            {
+              "title": "Copy to Clipboard",
+              "value": "copy"
+            }
+          ],
+          "required": false,
```

For backwards compatibility we can explicitly define default.

```suggestion
          "required": false,
          "default": "read",
```

---

Oh yeah, that's a good idea!